hyperspy/io_plugins/tvips.py

```diff
@@ -0,0 +1,347 @@
+            scan_dim = int(np.sqrt(total_scan_frames))
+            if not np.allclose(scan_dim, np.sqrt(total_scan_frames)):
+                raise ValueError("Scan was not square, please supply a scan_shape and start_frame.")
+            scan_shape = (scan_dim, scan_dim)
+            # there may be discontinuities which must be filled up
+            indices = _guess_scan_index_grid(record_idxs, scan_start_frame, scan_stop_frame)
+        # scan shape and start are provided
+        else:
+            total_scan_frames = scan_shape[0] * scan_shape[1]
+            indices = np.arange(scan_start_frame, scan_start_frame+total_scan_frames).reshape(scan_shape[0], scan_shape[1])
+
+        # with winding scan, every second column or row must be inverted
+        # due to hysteresis there is also a predictable offset
+        if winding_scan_axis is not None:
+            if winding_scan_axis in ["x", 0]:
+                indices[::2] = indices[::2][:, ::-1]
+                indices[::2] = np.roll(indices[::2], hysteresis, axis=1)
+            elif winding_scan_axis in ["y", 1]:
+                indices[:, ::2] = indices[:, ::2][::-1, :]
+                indices[:, ::2] = np.roll(indices[:, ::2], hysteresis, axis=0)
+            else:
+                raise ValueError("Invalid winding scan axis")
+        
+        with dask.config.set(**{'array.slicing.split_large_chunks': True}):
+            data_stack = data_stack[indices.ravel()]
+        data_stack = data_stack.reshape(*indices.shape, dimy, dimx)
+        units = ['nm', 'nm', DPU, DPU]
+        names = ['y', 'x', 'dy', 'dx']
+        # no scale information stored in the scan!
+        scales = [1, 1, SDP, SDP]
+        offsets = [0, 0, offsety, offsetx]
+        # Create the axis objects for each axis
+        dim = data_stack.ndim
+        axes = [
+            {
+                'size': data_stack.shape[i],
+                'index_in_array': i,
+                'name': names[i],
+                'scale': scales[i],
+                'offset': offsets[i],
+                'units': units[i], }
+            for i in range(dim)]
+        if lazy:
+            data_stack = data_stack.rechunk({0: "auto", 1: "auto", 2: None, 3: None})
+    else:
+        # we load as a regular image stack
+        units = ['s', DPU, DPU]
+        names = ['time', 'dy', 'dx']
+        times = all_metadata[0]["timestamp"] + all_metadata[0]["ms"]/1000
+        timescale = times[1] - times[0]
+        scales = [timescale, SDP, SDP]
+        offsets = [times[0], offsety, offsetx]
+        # Create the axis objects for each axis
+        dim = data_stack.ndim
+        axes = [
+            {
+                'size': data_stack.shape[i],
+                'index_in_array': i,
+                'name': names[i],
+                'scale': scales[i],
+                'offset': offsets[i],
+                'units': units[i], }
+            for i in range(dim)]
```

Comment:
Following on from the previous [comment ](https://github.com/hyperspy/hyperspy/pull/2781/files/a1bee7caf10dcac17a9785eb38a715bf594e3dc2#r705321351), a similar addition here:
```'navigate': True if not i else False```